benchmarks/distributed/rpc/ps/metrics/MetricsLogger.py

```diff
@@ -0,0 +1,92 @@
+from .CPUMetric import CPUMetric
+from .CUDAMetric import CUDAMetric
+
+
+class MetricsLogger:
+
+    CUDA = "cuda"
+    CPU = "cpu"
+
+    def __init__(self, rank, metric_class, overwrite_metrics=False):
```

Comment:
If `overwrite_metrics=True` was never used in this PR. Shall we remove it from this one and add it back when needed in the future?

Reply:
okay.